tools/introspection/source/introspection_app.cpp

```diff
@@ -771,13 +771,13 @@ void IntrospectionApp::runIntrospection(const iox::units::Duration updatePeriodM
         refreshTerminal();
 
         // Watch user input for updatePeriodMs
-        auto tWaitRemaining = std::chrono::milliseconds(updatePeriodMs.milliSeconds());
+        auto tWaitRemaining = std::chrono::milliseconds(updatePeriodMs.toMilliSeconds());
```

Comment:
not strictly related to the PR, but `updatePeriodMs` is wrong, this should be just `updatePeriod`. Could you also change this?

Reply:
Done.